cadl-tests/src/main/java/com/cadl/longrunning/LongRunningClient.java

```diff
@@ -40,7 +40,9 @@ public final class LongRunningClient {
      *
      * <pre>{@code
      * {
-     *     type: String (Optional)
+     *     id: String (Required)
+     *     name: String (Required)
+     *     type: String (Required)
```

Comment:
Javadoc might need some improvement.

id and name property should be read-only, hence not settable in request,

however fix might not be easy, as model had to be cached differently as request and response,

@haolingdong-msft

Reply:
Shall we remove read-only properties or having some wording indicating that it is read-only in request body?